extension/encoding/skywalkingencodingextension/config.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package skywalkingencodingextension // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/skywalkingencodingextension"
+import (
+	"fmt"
+)
+
+type Config struct {
+	Protocol string `mapstructure:"protocol"`
```

Comment:
Is this field really needed? It seems that there is no other possible values for it.

Reply:
mainly to consider whether there will be other possible in the future, but in the short term, no others, I can update and remove it.